### DISCUSSION TITLE
Supported Cams for Substream

### DISCUSSION THREAD

**Discussion by AceSG1** [mrlt8/docker-wyze-bridge#756] [Q&A]
Is there a list of supported cams? 

Thank you.

**Comment by mrlt8** [comment]
Wyze seems to have enabled the feature in the firmware released after September 2022. As of right now, these are some of the minimum firmware versions that are known to support a substream:

https://github.com/mrlt8/docker-wyze-bridge/blob/56d59ca2264261ca8e059e4b0d00084b0a3feb23/app/wyzecam/api_models.py#L48

The doorbell firmware probably supports a substream, but still needs some work to get it to actually work in the bridge. #742 

You can manually force a substream with: `SUBSTREAM_CAM_NAME=True`

**Reply by AceSG1** [reply]
So any cam that has a firmware version 4.9.9 or higher will have it enabled?

**Reply by mrlt8** [reply]
For v2 yes. It will need to be 4.36.10 or higher for the v3.

**Reply by AceSG1** [reply]
I have all v3 and there all at 4.36.10 and have
` SUBSTREAM=True`

But when I go to 
rtsp://xxx.xxx.x.xxx:8554/cam-name-sub It wont load (VLC). But If I do rtsp://xxx.xxx.x.xxx:8554/cam-name Itll load...
